app/models/person.rb

```diff
@@ -4,6 +4,8 @@ class Person < DirectoryObject
   validates :email, uniqueness: true, :allow_blank => true, :allow_nil => true, format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i }
   #validates :phone, uniqueness: false, presence: false, format: { with: /\A[)(\d\-x+ ]*\z/ }
   validate :phone_must_be_valid
+  validates :person_title, uniqueness: false, :allow_blank => true, :allow_nil => true
```

Comment:
Hmmm, I don't see a 'unique' validator http://guides.rubyonrails.org/active_record_validations.html, but uniqueness checks right before saving if the field about to be saved is unique